### PR TITLE
BurstDataFrame: bugfix for S1B

### PR DESCRIPTION
**Pull request by yunjunz**
BurstDataFrame.py: support S1B via `fnmatch.filter([seq])` while searching for burst info. Identified by @vbrancat.

### CODE REVIEW

File: src/BurstDataFrame.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def update(self, zipname):
         """
 
         zf = zipfile.ZipFile(zipname, 'r')
-        xmlpath = os.path.join('*SAFE','annotation', 's1a-iw{}-slc*xml'.format(self.swath))
+        xmlpath = os.path.join('*SAFE','annotation', 's1[ab]-iw{}-slc*xml'.format(self.swath))
```
Comment (vbrancat) [inline src/BurstDataFrame.py:97]: Sorry, I can't add a comment on the line of interest. Line 108 seems also to be hard-coded for S1A. I think the "relative orbit number" for S1B should be (if I am not wrong, I will try to add a reference to back up this equation)

**Relative Orbit Number = mod (Absolute Orbit Number orbit - 27, 175) + 1**

Comment (hfattahi) [reply src/BurstDataFrame.py:97]: Yes I agree. We need to fix that as well to be clear the equation is correct for S1A but is different for S1B:
S1-A: Relative Orbit Number = mod (Absolute Orbit Number orbit - 73, 175) + 1
S1-B: Relative Orbit Number = mod (Absolute Orbit Number orbit - 27, 175) + 1

There is a discussion here: https://forum.step.esa.int/t/sentinel-1-relative-orbit-from-filename/7042/20

@yunjunz will you fix this in this PR?

Comment (yunjunz) [reply src/BurstDataFrame.py:97]: Added now.

**Review by hfattahi — APPROVED** [review]
LGTM. Thanks Yunjun for the fix.

**Review by vbrancat — APPROVED** [review]
LGTM. Thanks a lot for this fix

**Pull request merged**